index.html

```diff
@@ -12,14 +12,24 @@
     <title>Document</title>
   </head>
   <body>
-<div class="banner">
+      <div class="social-links hide">
```

Comment:
- [ ] The mobile menu is missing from your mobile design screen. Kindly ensure to add a hamburger menu on the left side of the navigation for the mobile screen, which should display the mobile menu when clicked